Gems/Atom/RPI/Code/Source/RPI.Builders/Model/ModelExporterComponent.h

```diff
@@ -55,7 +55,7 @@ namespace AZ
 
                 AZStd::string_view m_relativeFileName;
                 AZStd::string_view m_extension;
-                const Uuid m_sourceUuid;
+                const Uuid m_sourceUuid={};
```

Comment:
Uuid default constructor doesn't zero the actual `char m_data[16]` array, so it is still filled with uninitialized values.

What should be used here is `Uuid::CreateNull()`